# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mine/install_static.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Makefile.orig	2005-04-14 14:10:06.917896488 -0400
+++ ./Makefile	2005-04-14 14:10:31.174208968 -0400
@@ -93,6 +93,7 @@
 
 install: all
 	install mine $(sysprefix)/bin/mine
+	install mine.static $(sysprefix)/bin/mine.static
 	install tarbz2gem.sh $(prefix)/sbin/tarbz2gem
 	install rocket.sh $(prefix)/sbin/rocket
 ifeq ($(USE_GAS), 1)
